src/FSharp.Data.GraphQL.Shared/TypeSystem.fs

```diff
@@ -546,6 +547,9 @@ and TypeDef<'Val> =
 and InputDef =
     interface
         inherit TypeDef
+
+        /// Serializes the associated .NET CLR type into a GQL value
+        abstract member Serialize : obj -> Ast.Value
```

Comment:
We need access to `Serialize` in places where we have an erased `InputDef`, not an `InputDef<'Val>`. 

We could have two serialize fields (one on `InputDef` and one on `InputDef<'Val>`) but perhaps this would be more trouble than it's worth?